Thumbprint/Targets/Playground/Assets/Assets.xcassets/niccage.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "niccage.jpg",
```

Comment:
Just to be safe, might be good to replace this with an image from something like Unsplash (where everything is freely usable) and then include a link to the license (e.g., https://unsplash.com/license)